includes/theme-support/class-wc-twenty-eleven.php

```diff
@@ -6,6 +6,8 @@
  * @package WooCommerce/Classes
  */
 
+use Automattic\WooCommerce\Theming\ThemeSupport;
```

Comment:
Agree that namespaces should be nouns, but isn't "Theming" also a noun? As in "anything related to themes". If we use "Theme" that would be confusing as it seems to imply that the folder/namespace actually contains a theme.

Reply:
Well, `Theming` is the process by which you create a `Theme`. I actually really like `ThemeManagement` because we have a LOT of theme related code that could live there!